src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -484,33 +488,36 @@ private CodeRefactoring FilterOnUIThread(CodeRefactoring refactoring, Workspace
             {
                 foreach (var fix in codeFixes)
                 {
-                    SuggestedAction suggestedAction;
-                    if (fix.Action.NestedCodeActions.Length > 0)
+                    var suggestedAction = GetSuggestedAction(fix.Action, fix);
+                    AddFix(fix, suggestedAction, map, order);
+                }
+
+                return;
+
+                // Local functions
+                SuggestedAction GetSuggestedAction(CodeAction action, CodeFix fix)
```

Comment:
consider making static if possible.

Reply:
That leads to 7-8 additional parameters, I'll keep it as is.